server/src/main/java/com/lunchtime/models/Event.java

```diff
@@ -0,0 +1,56 @@
+package com.lunchtime.models;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.net.URI;
+import java.util.Date;
+
+
+@ToString
+@Setter
+@Getter
+@Entity
+@Table(name = "event")
+public class Event {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @NotNull
+    @Column(name = "restaurant_id")
+    private Long restaurantId;
+
+    @NotNull
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "date")
+    private Date date;
+
+    @NotNull
+    @Column(name = "image")
+    private URI image;
+
+    @NotBlank
+    @Size(min=6, max=255)
```

Comment:
You should disable rule `WhitespaceAround` in checkstyle or update your code as it destroys building process
```
lunch-time-server | [ERROR] /app/src/main/java/com/lunchtime/models/Event.java:40:14: WhitespaceAround: '=' is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3) [WhitespaceAround]
lunch-time-server | [ERROR] ...
```